content/en/agent/basic_agent_usage/_index.md

```diff
@@ -150,7 +150,9 @@ When the Agent is running, use the `datadog-agent launch-gui` command to open th
 | [Windows 64-bit][10]                            | Windows 7+                                        |
 | [Windows Azure Stack HCI OS][10]                | All Versions                                      |
 
-**Note**: [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+**Notes**: 
+- [Source][11] install may work on operating systems not listed here and is supported on a best effort basis.
+- Datadog Agent v7+ supports Windows Server 2008 R2+ with the most recent Windows updates installed. There is also a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2+.
```

Comment:
Unless I'm missing something this note only applies to Windows Server 2008 R2 specifically, not later Windows versions:

```suggestion
- Datadog Agent v7+ supports Windows Server 2008 R2 with the most recent Windows updates installed. There is also a [known issue with clock drift and Go][12] that affects Windows Server 2008 R2.
```